src/Data/List/Properties.agda

```diff
@@ -758,6 +758,11 @@ length-take zero    xs       = refl
 length-take (suc n) []       = refl
 length-take (suc n) (x ∷ xs) = cong suc (length-take n xs)
 
+-- Taking and dropping from an empty list do nothing.
```

Comment:
This comment should only reference `taking`, likewise the one below only `dropping`.